app/code/Magento/Inventory/Model/ReservationBuilderInterface.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Inventory\Model;
+
+use Magento\InventoryApi\Api\Data\ReservationInterface;
+
+/**
+ * Used to instantiate ReservationInterface objects
+ *
+ * @see ReservationInterface
```

Comment:
this interface should be marked as `@api`
because it's supposed to be called from the code of business logic